examples/plot_tensorflow.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+=================================
+An example for tensorflow
+=================================
+"""
+# %%
+
+import pandas as pd
+from sklearn.datasets import fetch_openml
+from sklearn.metrics import accuracy_score, precision_score, recall_score
+import scikeras
+import warnings
+from tensorflow import get_logger
+import numpy as np
+from scikeras.wrappers import KerasClassifier

```

Comment:
Are these actually available in the doc build? I thought that TensorFlow was only installed for the `otherpackages` tests

Reply:
Actually yes, Tensorflow is not aligning with the dependencies which are already there in `requirements-dev.txt`.